sample/GoogleIO17/sponsors

```diff
@@ -1 +1,155 @@
-[]
\ No newline at end of file
+[
```

Comment:
@niranjan94 
i was told by mario in case of mozilla all hands to list all products of mozilla as sponsors. i did the same over here. should i remove this then?